testinfra/modules/file.py

```diff
@@ -228,9 +230,7 @@ def md5sum(self):
 
     @property
     def sha256sum(self):
-        return self.check_output(
-            "sha256sum %s | cut -d ' ' -f 1", self.path)
-
```

Comment:
You'll need it for the BSD's too if I'm not mistaken. sha256 is gnu/coreutils which the BSD's don't ship with afaik.